src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/AlleleFractionSegmenter.java

```diff
@@ -0,0 +1,93 @@
+    private AllelicBiasParameters biasParameters;
+
+    private static final double MIN_MINOR_ALLELE_FRACTION = 0.0;
+    private static final double MAX_MINOR_ALLELE_FRACTION = 0.5;
+
+    /**
+     * Initialize the segmenter with its data and panel of normals, giving equal weight to a set of evenly-spaced
+     * hidden minor allele fraction values.
+     *
+     * @param initialNumStates  A liberal estimate of the number of hidden minor allele fraction values to
+     *                          include in the model.  Hidden states are pruned as the model is learned.
+     * @param acc               The {@link AllelicCountCollection} data attached to this segmenter
+     * @param allelicPoN        The {@link AllelicPanelOfNormals} attached to this segmenter
+     */
+    public AlleleFractionSegmenter(final int initialNumStates, final AllelicCountCollection acc,
+                                   final AllelicPanelOfNormals allelicPoN) {
+        super(initialNumStates, acc.getCounts().stream().map(c -> c.getInterval()).collect(Collectors.toList()), acc.getCounts());
+        this.allelicPoN = Utils.nonNull(allelicPoN);
+    }
+
+    /**
+     * evenly-spaced minor allele fractions going from 1/2 to 0
+     * @param K the initial number of hidden states
+     */
+    @Override
+    protected void initializeHiddenStateValues(final int K) {
+        hiddenStateValues = IntStream.range(0, K).mapToDouble(n ->  ((double) K - n) / (2*K)).toArray();
+    }
+
+    @Override
+    protected void initializeAdditionalParameters() {
+        biasParameters = new AllelicBiasParameters(1.0, 1e-2, 3e-2);
+    }
+
+    @Override
+    protected ClusteringGenomicHMM<AllelicCount> makeModel() {
+        return new AlleleFractionHiddenMarkovModel(hiddenStateValues, weights, memoryLength, allelicPoN, biasParameters);
+    }
+
+    @Override
+    protected void relearnAdditionalParameters(final ExpectationStep eStep) {
+        final Function<AllelicBiasParameters, Double> emissionLogLikelihood = params -> {
+            double logLikelihood = 0.0;
+            for (int position = 0; position < positions.size(); position++) {
+                for (int state = 0; state < weights.length; state++) {
+                    final double eStepPosterior = eStep.pStateAtPosition(state, position);
+                    logLikelihood += eStepPosterior < NEGLIGIBLE_POSTERIOR_FOR_M_STEP ? 0 :eStepPosterior
+                            * AlleleFractionHiddenMarkovModel.logEmissionProbability(data.get(position), hiddenStateValues[state], params, allelicPoN);
+                }
+            }
+            return logLikelihood;
+        };
+
+        final Function<Double, Double> meanBiasObjective = mean -> emissionLogLikelihood.apply(biasParameters.copyWithNewMeanBias(mean));
+        final double newMeanBias = OptimizationUtils.quickArgmax(meanBiasObjective, 0, AlleleFractionInitializer.MAX_REASONABLE_MEAN_BIAS, biasParameters.getMeanBias());
```

Comment:
see note elsewhere about `quickArgmax`.  if you replace with an `argmax` method that allows specification of the tolerance, extract it accordingly here and below